docs/integrations/sources/instagram-migrations.md

```diff
@@ -1,5 +1,70 @@
 # Instagram Migration Guide
 
+## Upgrading to 3.0.0
+
+Connector will be upgraded to API v18. Affected Streams and their corresponding changes are listed below:
+
+- `Media Insights`
+  
+  Old metric will be replaced with the new ones, refer to the [IG Media Insights](https://developers.facebook.com/docs/instagram-api/reference/ig-media/insights#metrics) for more info.
+
+  | Old metric                 | New metric         |
+  |----------------------------|--------------------|
+  | carousel_album_engagement  | total_interactions |
+  | carousel_album_impressions | impressions        |
+  | carousel_album_reach       | reach              |
+  | carousel_album_saved       | saved              | 
+  | carousel_album_video_views | video_views        |
+  | engagement                 | total_interactions |
+
+:::note
+
+You may see different results: `engagement` includes likes, comments and saves count while `total_interactions` includes likes, comments, saved and shares count.
```

Comment:
You may see different results: `engagement` count includes likes, comments, and saves while `total_interactions` count includes likes, comments, and saves, as well as shares.